admission-controller/values.schema.json

```diff
@@ -0,0 +1,1002 @@
+{
```

Comment:
Did you generate this manually or using a tool? I don't see a `$schema` field, so I'm curious as to how this was created. Looks great!

Reply:
I used a tool and removed it and the linting seemed to work still. Is that field necessary?

Reply:
It's used to specify the schema version of the JSON file itself. I'd recommend keeping it.

What tool did you use?

Reply:
Ah that makes sense, I'll add it back in. I used a yaml to json tool and then https://www.jsonschema.net/home.